blob/service.go

```diff
@@ -198,10 +198,10 @@ func (s *Service) getByCommitment(
 	}
 
 	var (
-		rawShares = make([]share.Share, 0)
+		rawShares = make([]shares.Share, 0)
```

Comment:
is `raw` intentional or is this a typo of `row`?

Reply:
Intentional